apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx

```diff
@@ -413,11 +420,15 @@ const Modal: React.FC<MultipleFiltersModalProps> = ({
     const handleSaveRef = useRef(null)
     const [form] = Form.useForm()
 
-    const { selectedFiltersTemplate, setSelectedFiltersTemplate } = useMultipleFilterContext()
+    const { selectedFiltersTemplate, setSelectedFiltersTemplate, loggerConverters } = useMultipleFilterContext()
     const [openedFiltersTemplate, setOpenedFiltersTemplate] = useState(
         isEqualSelectedFiltersTemplateAndFilters(selectedFiltersTemplate, filters) ? selectedFiltersTemplate : null
     )
 
+    const { logEvent } = useTracking()
+
+    const eventName = eventNamePrefix ? `${eventNamePrefix}FilterModalClickSubmit` : ''
```

Comment:
By default if you don't specify custom eventName, you should get auto generated one from `useTracking` hook. Method name is `getEventName`

Reply:
It is necessary?
If this is done, then this event will have some common name and will possibly be duplicated

Reply:
Overused default name `TicketClickIndex` (or something similar)

Reply:
Now the logic is such that if a prefix is not specified, then there is no `eventName` and statistics will not be sent

Reply:
added `getEventName`. And added event name in payload